packages/webapp/src/hooks/useConnections.tsx

```diff
@@ -38,6 +39,15 @@ export function useConnections(queries: GetConnections['Querystring']) {
     return { loading, error: error?.json, data, hasNext, offset: size, setOffset: setSize, mutate };
 }
 
+export function invalidateConnections(cache: Cache, mutate: ReturnType<typeof useSWRConfig>['mutate']) {
```

Comment:
note this is more than invalidate, this is nuking the cache which will directly trigger a fetch call (vs soft cache invalidation which only triggers on next page change)

Reply:
Yeah, I tried to only mutate it with an empty value so that it would invalidate, I cold never get that to work with `swr/infinite`. Renamed the function to `clearConnectionsCache` in 48b412cc